merlin/datasets/entertainment/movielens/1m-raw/ratings/schema.pbtxt

```diff
@@ -0,0 +1,68 @@
+feature {
+  name: "userId"
+  type: INT
+  int_domain {
+    name: "userId"
+    max: 6041
+    is_categorical: true
+  }
+  annotation {
+    tag: "categorical"
+    tag: "item_id"
+    tag: "item"
+    extra_metadata {
+      type_url: "type.googleapis.com/google.protobuf.Struct"
+      value: "\n\021\n\013num_buckets\022\002\010\000\n\033\n\016freq_threshold\022\t\021\000\000\000\000\000\000\000\000\n\025\n\010max_size\022\t\021\000\000\000\000\000\000\000\000\n\030\n\013start_index\022\t\021\000\000\000\000\000\000\000\000\n1\n\010cat_path\022%\032#.//categories/unique.userId.parquet\nG\n\017embedding_sizes\0224*2\n\030\n\013cardinality\022\t\021\000\000\000\000\000\231\267@\n\026\n\tdimension\022\t\021\000\000\000\000\000@j@\n\034\n\017dtype_item_size\022\t\021\000\000\000\000\000\000P@\n\r\n\007is_list\022\002 \000\n\017\n\tis_ragged\022\002 \000"
+    }
+  }
+}
+feature {
+  name: "movieId"
+  type: INT
+  int_domain {
+    name: "movieId"
+    max: 3707
+    is_categorical: true
+  }
+  annotation {
+    tag: "categorical"
+    tag: "item_id"
+    tag: "item"
+    extra_metadata {
+      type_url: "type.googleapis.com/google.protobuf.Struct"
+      value: "\n\021\n\013num_buckets\022\002\010\000\n\033\n\016freq_threshold\022\t\021\000\000\000\000\000\000\000\000\n\025\n\010max_size\022\t\021\000\000\000\000\000\000\000\000\n\030\n\013start_index\022\t\021\000\000\000\000\000\000\000\000\n2\n\010cat_path\022&\032$.//categories/unique.movieId.parquet\nG\n\017embedding_sizes\0224*2\n\030\n\013cardinality\022\t\021\000\000\000\000\000\366\254@\n\026\n\tdimension\022\t\021\000\000\000\000\000\000d@\n\034\n\017dtype_item_size\022\t\021\000\000\000\000\000\000P@\n\r\n\007is_list\022\002 \000\n\017\n\tis_ragged\022\002 \000"
+    }
+  }
+}
+feature {
+  name: "rating"
+  type: INT
+  int_domain {
+    name: "rating"
+    max: 6
+    is_categorical: true
+  }
+  annotation {
+    tag: "categorical"
```

Comment:
Should we tag this as target?

Reply:
This schema represents the raw data for unittests. The raw data does not provide any tags. I wanted to use as few tags as possible. Actually, we should remove the tags from all columns, but then the data generator did not work.